web/api/v1/api_test.go

```diff
@@ -1501,6 +1501,7 @@ func testEndpoints(t *testing.T, api *API, tr *testTargetRetriever, es storage.E
 								Type:   "recording",
 							},
 						},
+						Limit: 1,
```

Comment:
This should be a `0`, not a `1`, because we don't set this field in the API tests.

It should also come after the `Interval` field at the top of the `RuleGroup` struct.